crates/compilers/src/compilers/mod.rs

```diff
@@ -201,16 +202,18 @@ pub trait CompilationError:
 /// Output of the compiler, including contracts, sources and errors. Currently only generic over the
 /// error but might be extended in the future.
 #[derive(Debug, Serialize, Deserialize)]
-pub struct CompilerOutput<E> {
+pub struct CompilerOutput<E, C> {
     #[serde(default = "Vec::new", skip_serializing_if = "Vec::is_empty")]
     pub errors: Vec<E>,
-    #[serde(default)]
-    pub contracts: FileToContractsMap<Contract>,
+    #[serde(default = "BTreeMap::new")]
+    pub contracts: FileToContractsMap<C>,
     #[serde(default)]
     pub sources: BTreeMap<PathBuf, SourceFile>,
+    #[serde(default)]
+    pub metadata: BTreeMap<String, String>,
```

Comment:
are these really just strings or are those json objects?

Reply:
Changed value to json: https://github.com/foundry-rs/compilers/pull/224/commits/c2a6c688ae8048c3b02db8f3268c72017893000f